.travis.yml

```diff
@@ -47,11 +47,14 @@ jobs:
       env:
         ARCH: s390x
         ARROW_CI_MODULES: "CPP"
-        ARROW_FLIGHT: "OFF"
+        ARROW_FLIGHT: "ON"
         ARROW_GANDIVA: "OFF"
         ARROW_PARQUET: "OFF"
         PARQUET_BUILD_EXAMPLES: "OFF"
         PARQUET_BUILD_EXECUTABLES: "OFF"
+        cares_SOURCE: "BUNDLED"
+        Protobuf_SOURCE: "BUNDLED"
+        gRPC_SOURCE: "BUNDLED"
```

Comment:
Could you sort `env` list alphabetically?

Reply:
Sure, just a simple question. Should we sort by case insensitive order?

Reply:
Case sensitive:

```diff
diff --git a/.travis.yml b/.travis.yml
index de51382c0..6e2aee94f 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -50,13 +50,13 @@ jobs:
         ARROW_FLIGHT: "ON"
         ARROW_GANDIVA: "OFF"
         ARROW_PARQUET: "OFF"
+        DOCKER_IMAGE_ID: ubuntu-cpp
         PARQUET_BUILD_EXAMPLES: "OFF"
         PARQUET_BUILD_EXECUTABLES: "OFF"
-        cares_SOURCE: "BUNDLED"
         Protobuf_SOURCE: "BUNDLED"
-        gRPC_SOURCE: "BUNDLED"
-        DOCKER_IMAGE_ID: ubuntu-cpp
         UBUNTU: "20.04"
+        cares_SOURCE: "BUNDLED"
+        gRPC_SOURCE: "BUNDLED"
   allow_failures:
     - arch: s390x
```

(Oh, I put `DOCKER_IMAGE_ID` wrong place...)